pkg/reconciler/channel/channel.go

```diff
@@ -131,7 +165,7 @@ func (r *Reconciler) reconcileBackingChannel(ctx context.Context, channelResourc
 		return nil, err
 	}
 	logging.FromContext(ctx).Debug("Found backing Channel", zap.Any("backingChannel", backingChannelObjRef))
-	channelable, ok := backingChannel.(*duckv1beta1.Channelable)
+	channelable, ok := backingChannel.(*duckv1alpha1.ChannelableCombined)
```

Comment:
I think this here is a bug too btw. on L171 we return nil, err which is not right, since there's no err.

So, you might change the L171 to:
return nil, fmt.Errorf("failed to convert to channelable object %+v", backingChannel)

I wonder if that's causing the panic you're seeing?

Reply:
the logging on line 170 also makes use of `err` as well, I'll drop that line too

Reply:
Super duper. Thanks!